src/App/App.js

```diff
@@ -69,8 +69,12 @@ class App extends Component {
     }
   };
   render() {
+    const logged = this.state.loggedIn ? (
+      <Redirect to="/" />
+    ) : ( <Login toggleLogin={this.toggleLogin} />
+    )
```

Comment:
Still would like to have this exist somewhere else, feels a little weird to have it in the render like this.